src/cmdlib.sh

```diff
@@ -537,7 +539,10 @@ EOF
     fi
 
     # this is the command run in the supermin container
-    echo "$@" > "${tmp_builddir}"/cmd.sh
+    # we hardcode a umask of 0022 here to make sure that composes are run
+    # with a consistent value, regardless of the environment
+    echo "umask 0022" > "${tmp_builddir}"/cmd.sh
```

Comment:
Note `runvm` is used for other things so this doesn't affect just `rpm-ostree compose`. But I think we should just use 0022 for everything unless there's a good reason not to.